### DISCUSSION TITLE
Install Redash on Linux

### DISCUSSION THREAD

**Discussion by saurabhhangotra** [getredash/redash#6602] [General]
I was using AMI for redash installation previously but now its not working, I tried to install using docker but its not working. Anyone can share me details how can i install in linux system

I have followed below steps:-

created .env and added keys

I used below docker image  and ran "docker-compose run --rm server create_db" and 
"docker-compose up -d "

`version: '3'
services:
  server:
    image: redash/redash:latest
    depends_on:
      - postgres
    ports:
      - "5000:5000"
    environment:
      REDASH_WEB_WORKERS: 4
  scheduler:
    image: redash/redash:latest
    depends_on:
      - postgres
    environment:
      QUEUES: "celery"
  worker:
    image: redash/redash:latest
    depends_on:
      - postgres
    environment:
      QUEUES: "celery"
  redis:
    image: "redis:latest"
  postgres:
    image: postgres:9.6.2
    environment:
      POSTGRES_PASSWORD: "xxx-xxx-xxx"
`

I ran successfully 
![image](https://github.com/getredash/redash/assets/76439768/ecd3f09b-61f1-492c-8901-12b558411b00)

But still i don't see anything

![image](https://github.com/getredash/redash/assets/76439768/52548c2b-11b3-4154-8d10-4d4dcea5e655)

**Comment by guidopetri** [comment]
It seems you're using two different IPs for your machine. Have you tried the `172.x.y.z` IP?

Did you open the ports to the EC2 machine to allow for outside access, or somehow add yourself to the VPC?

**Reply by saurabhhangotra** [reply]
Yes i did enabled EC2 port 

![image](https://github.com/getredash/redash/assets/76439768/e8a4f89d-431e-44d2-8285-15e6a66ea088)

Also using public ip to access it
![image](https://github.com/getredash/redash/assets/76439768/0b418fc4-ff86-4ae3-b7f9-8c9145a43313)

**Reply by guidopetri** [reply]
Ok, I see the issue. You're forwarding the postgres container ports, and not the redash container ports.
